README.md

```diff
@@ -1,33 +1,71 @@
 # Tagging Tracker
 The CRT Graffiti Abatement Pilot Project is a proactive approach to removing graffiti along the Prospect Avenue corridor of Kansas City. The Tagging Tracker Tool will provide the infrastructure to track the occurrence of graffiti and its abatement. The tool is an application that allows the team to take pictures graffiti, geolocate those picture, and store information about each building with graffiti and each graffiti tag. This data will help the project team evaluate the effectiveness of our methods and help to build a case for future funding.
- 
+
 ## Why Remove Graffiti?
 “Do not invest here. This neighborhood is unsafe.” Graffiti may not say these words explicitly, but that is the message often sent when taggers or local gangs deface buildings, instantly reducing their value, and ultimately the value of the entire community. Graffiti can mark gang turf and can claim credit for crime, thereby setting off of a cycle of retaliation. Graffiti sends a signal to residents, criminals, and potential businesses that this area is not cared for and is not safe. Graffiti abatement is a critical catalyst for both improved safety and economic development.
- 
+
 A number of business owners along Prospect Avenue feel frustrated and almost defeated by the amount of repeated graffiti on their building. One business owner even closed their store because of the repeated tagging on their building. Local businesses experience a drain on their resources when they have to pay fines from a codes violation and on contractors to remove graffiti. One business owner has spent $3,000 over the last year on painting over graffiti, only to have it return months later. To make matters worse, a large number of buildings along Prospect are vacant or abandoned, leaving them in a no-man’s land of graffiti abatement until a concerned citizen or neighborhood association takes it upon themselves to improve the property.
- 
+
 ## Stakeholders
 In 2015, the Community Resource Team (CRT) developed out of a 3-year U.S. Department of Justice Byrne Criminal Justice Innovation (BCJI) grant and is a cross-sector collaboration of community leaders, social services providers, government institutions and law enforcement that jointly identify and address community safety issues in the Prospect Avenue corridor. CRT works to create a safer community and build collective efficacy within the 2.15 square mile area from Paseo Boulevard to Indiana Avenue and from 25th Street to 39th Streets, an area identified because of the prevalence of violent crime.
- 
+
 The CRT Graffiti Abatement Pilot Project area will stretch from 18th Street to 39th, 3-4 blocks east and west of Prospect Avenue. This project will indirectly serve residents and business owners in the area.
- 
+
 ## Approach / Roadmap:
- 
+
 
 ## Team:
 * Amanda Wilson
 * Jake LaCombe
 * Aaron McRuer
 * Kol Kheang
+* Fred Lawl
 
 Code for KC Slack Channel: #taggingtracker
 
 
 ## Developers Notes
```

Comment:
I'm not sure about calling it developer notes. Should we instead call it Install or something like that?